pkg/network/ebpf/c/runtime/tracer.c

```diff
@@ -689,6 +733,67 @@ int socket__http_filter(struct __sk_buff* skb) {
     return 0;
 }
 
+SEC("kprobe/ip_route_output_flow")
+int kprobe__ip_route_output_flow(struct pt_regs* ctx) {
+    struct net *net = (struct net*) PT_REGS_PARM1(ctx);
+    struct flowi4* fl4 = (struct flowi4*) PT_REGS_PARM2(ctx);
```

Comment:
Is IPv6 support coming in a later PR?

Reply:
Yes, we have a card for it in the backlog.